src/ontology/edits/oeo-physical.omn

```diff
@@ -8577,6 +8582,23 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1768",
              and (<http://purl.obolibrary.org/obo/RO_0000056> some OEO_00010141))
     
     
+Class: OEO_00010471
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A non-energy use technology is a technology that describes among other things how to combine non-energy use of energy carriers with other processes and artificial objects or other material entities in a specific way."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A non-energy use technology is a technology that describes among other things how to combine the consumption of energy carriers for non-energy use with other processes and artificial objects or other material entities in a specific way."@en,
```
I find it hard to create a good definition and hope my suggestion makes it better.